src/reducer.js

```diff
@@ -118,6 +128,20 @@ export const actionReducers = {
   },
 
   /**
+   * Update the bunsen model
+   * @param {State} state - state to update
+   * @param {Action} action - action
+   * @returns {State} - updated state
+   */
+  [CHANGE_VIEW]: function (state, action) {
```

Comment:
This is ES6 dynamic property declaration (or whatever it's called). It ensures the key of this function always exactly matches the value of the constant [CHANGE_VIEW] in case we change the actual string value of actions in the future. E.g. one convention I've seen uses `<component-name>/<ACTION>` which is useful when combining multiple reducers.